src/saved_search_notification/saved_search_notification.entity.ts

```diff
@@ -11,19 +16,22 @@ export class SavedSearchNotification {
     @PrimaryGeneratedColumn()
     id: number;
 
-    @Column()
-    emailAddress: string;
-
-    @Column()
-    savedSearchName: string;
-
     @Column()
     resultsUri: string;
 
+    @ManyToOne(() => User, (user) => user.savedSearchNotifications, {
+        eager: true,
```

Comment:
Any reason why we are eagerly fetching here?

If we are accessing the user object every time we access the saved search notification then that is fine however we will to make sure user is indexed properly and that we have tested this at scale

Reply:
We only use this table in notifications.service where access user each time. Will have a look re indexing

Reply:
Added 6k users locally and it's taking 110ms

<img width="412" alt="Screenshot 2023-10-11 at 10 42 34" src="https://github.com/cabinetoffice/gap-find-backend/assets/135222889/71285b04-9c48-4815-9ff6-1caf2f339e8d">